crates/p2p/src/events.rs

```diff
@@ -1,18 +1,10 @@
-//! Events emitted from P2P.
+/// Events emitted from P2P that needs to be handled by the user.
```

Comment:
This is a module-level docstring, hence should be prefixed with `//!`:

```suggestion
//! Events emitted from P2P that needs to be handled by the user.
```